.semaphore/pre-release-prepare.yml

```diff
@@ -0,0 +1,90 @@
+version: v1.0
+name: Paprika pre-release prepare
+agent:
+  machine:
+    type: e1-standard-4
```

Comment:
I don't know now that we update our machine to `e1-standard-4` would be necessary to do 

```js
- sudo swapoff -a && sudo fallocate -l 2G /swapfile && sudo mkswap /swapfile && sudo swapon /swapfile

```